catalog.html

```diff
@@ -56,23 +56,201 @@
         </div>
     </header>
 
+    <section class="blog-intro">
+        <div class="blog-intro__wrapper">
+            <h1 class="blog-intro__title">Explore catalogs</h1>
+            <ul class="breadcrumbs">
+                <li href="#" class="breadcrumbs__item"><a href="">Home</a></li>
+                <li href="#" class="breadcrumbs__item">Catalog</li>
+            </ul>
+        </div>
+    </section>
 
-    <main class="main main--two-col">
+    <main class="main main--two-col main--catalog">
         <aside class="sidebar">
-            <details class="faq__item">
-                <summary class="faq__question">What are the NFTs?</summary>
-                <p class="faq__answer">Urna vitae erat et lacus, consectetur ac nulla vestibulum lobortis. Nulla dapibus urna volutpat venenatis, risus faucibus</p>
-            </details>
+            <form class="form-sidebar" action="#" method="get">
+                <details class="faq__item">
+                    <summary class="faq__question">Clothing</summary>
+                    <div class="faq__answer">
+                        <label class="form-checkbox">
```

Comment:
There is too big clickable area. The checkbox and text should be clickable only.
<img width="359" alt="Screenshot 2024-12-16 at 01 04 01" src="https://github.com/user-attachments/assets/fd3ff68e-1561-4e69-8a30-08a389a70b7b" />

Reply:
done